task-02-damianti/6.2.two_thousand_run.py

```diff
@@ -0,0 +1,32 @@
+
+import time
+
+
+def timer(f, *args, **kwargs):
+    """The timer function takes the following parameters:
+
+    :param f: the function to be measured
+    :param args: any positional arguments that f requires
+    :param kwargs: any keyword arguments that f requires
+    :return: the execution time of the function as a floating-point number.
+    """
+    start = time.time()
+
+    f(*args, **kwargs)
+
+    end = time.time()
+    return end - start
+
+
+if __name__ == "__main__":
```

Comment:
its better practice to put the code underneath the if __name__ == "__main__" inside a main() function.